### ISSUE TITLE
Add liblinear

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/gateplugin-LearningFramework#42]
There is a java implementation of liblinear there: https://github.com/bwaldvogel/liblinear-java

May be interesting to include for classification.

**Comment by domrout** [comment]
Is there any advantage to doing that through liblinear-java since scikit-learn already supports it?

**Comment by ianroberts** [comment]
It'd be a pure Java solution rather than relying on a native installation of scikit-learn.

**Comment by domrout** [comment]
True, but I'd say it's still a low priority until we actually have a problem with this. So the later tag is good 👍

**Comment by johann-petrak** [comment]
Accidently closed by wrong commit message.
